packages/playground/alias/__tests__/alias.spec.ts

```diff
@@ -1,4 +1,14 @@
-import { editFile, getColor, untilUpdated } from '../../testUtils'
+import {
+  editFile,
+  getColor,
+  untilUpdated,
+  uvuSetup,
+  uvuReset
+} from '../../testUtils'
+import { test } from 'uvu'
+
+test.before(async () => await uvuSetup(__dirname))
```

Comment:
__dirname in esm?

Reply:
I guess `uvu` runs the tests in CJS if no `type: module` in `package.json`

Reply:
Likely jiti polyfilled it so it worked, but I agree we should avoid CJS stuff in ESM (since it's using `import`s and TS is ESM by default). Maybe using `tsm` would help catch these quirks easily.